js/header_dialog.js

```diff
@@ -101,6 +94,21 @@ function HeaderDialog(dialog, onSave) {
         {name:'vbat_sag_compensation'        , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'4.3.0', max:'999.9.9'},
         {name:'gyro_to_use'                  , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'4.3.0', max:'999.9.9'},
         {name:'dynamic_idle_min_rpm'         , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'4.3.0', max:'999.9.9'},
+        {name:'ff_transition'                , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'3.5.0', max:'999.9.9'},
```

Comment:
3.5 or 4.3 ?

Reply:
Transition first came in back in 3.5.  It was called D weight back then, from memory.